### DISCUSSION TITLE
Lighter dark mode background

### DISCUSSION THREAD

**Discussion by he3als** [pelican-dev/panel#398] [Ideas]
Maybe the dark mode background could be less black? The current background is okay, but something lighter could be less harsh.

### Current
![image](https://github.com/pelican-dev/panel/assets/65787561/df024146-cf82-4a41-a345-abb819769acc)

### Proposal
![image](https://github.com/pelican-dev/panel/assets/65787561/1217d386-b13a-47ea-9cea-8fd7c46bf9ba)

**Comment by notAreYouScared** [comment]
This will be able to be done via themes, when they come.
